addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/NestBindingConstants.java

```diff
@@ -20,22 +20,22 @@
 
     public static final String BINDING_ID = "nest";
 
-    /** The url to use to connect to nest with. */
+    /** The url to use to connect to Nest with. */
     public static final String NEST_URL = "https://developer-api.nest.com/";
 
-    /** The url to get the access token when talking to nest. */
+    /** The url to get the access token when talking to Nest. */
     public static final String NEST_ACCESS_TOKEN_URL = "https://api.home.nest.com/oauth2/access_token";
 
-    /** The url to get set values on the thermostat when talking to nest. */
+    /** The url to get set values on the thermostat when talking to Nest. */
     public static final String NEST_THERMOSTAT_UPDATE_URL = NEST_URL + "/devices/thermostats/";
```

Comment:
`NEST_URL = "https://developer-api.nest.com/"`
`NEST_THERMOSTAT_UPDATE_URL = NEST_URL + "/devices/thermostats/"`

So the resulting URL has 1 forward slash too many:  `https://developer-api.nest.com//devices/thermostats/`

The same holds for the other URLs that depend on NEST_URL.

Reply:
Done, I prefer to have them start with a slash so removed it from the `NEST_URL`